vuln-reach/Cargo.toml

```diff
@@ -2,6 +2,11 @@
 name = "vuln-reach"
 version = "0.1.0"
 edition = "2021"
+description = "A library for building tools to determine if vulnerabilities are reachable in a code base."
```

Comment:
Seems rather verbose, especially because things like "a library for building tools" are just filler.

"Code reachability path analysis".